### DISCUSSION TITLE
[Unstable] Loop sounds triggered by statuseffect cannot play correctly

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14230] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Loop sound will stop immediately after playing and keep cycling this.

https://github.com/FakeFishGames/Barotrauma/assets/73634685/236d4189-6f7c-472f-9074-cf2f0ded1e5f

### Reproduction steps

Equip respawn diving suit with less than 5% condition

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.100.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested to be working correctly now. Fix coming in the next update. Closing discussion.
